statline_bq/utils.py

```diff
@@ -6,12 +6,14 @@
 from datetime import datetime
 from shutil import rmtree
 from tempfile import gettempdir
+import xml.etree.ElementTree as ET
 
 import ndjson
 from dask.bag import Bag as DaskBag
 import dask.bag as db
```

Comment:
same as comment below
`import dask.bag as db`

Reply:
The `DaskBag` here was used for TypeChecking in the deprecated `get_odata` function, and can be removed.